suit_generator/cmd_sign.py

```diff
@@ -0,0 +1,321 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+"""CMD_SIGN CLI command entry point."""
+
+import json
+import uuid
+import logging
+import cbor2
+import importlib.util
+import sys
+import os
+from pathlib import Path
+from suit_generator.suit_sign_script_base import (
+    SuitEnvelopeSignerBase,
+    SignatureAlreadyPresentActions,
+    SuitSignAlgorithms,
+)
+from suit_generator.exceptions import GeneratorError
+from argparse import RawTextHelpFormatter
+
+SIGN_SINGLE_LEVEL_CMD = "single-level"
+SIGN_RECURSIVE_CMD = "recursive"
+
+log = logging.getLogger(__name__)
+
+SIGN_CMD = "sign"
+
+
+def _import_module_from_path(module_name, file_path):
```

Comment:
Similar functions can be found across multiple files in this PR - maybe it could be extracted?

Reply:
It was just an observation, it is not crucial.